frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -495,15 +494,16 @@ export default class ProblemDetails extends Vue {
   }
 
   onRunDetails(guid: string): void {
+    this.currentPopupDisplayed = PopupDisplayed.RunDetails;
     this.$emit('show-run', {
       request: {
         guid,
+        hash: `#problems/show-run:${guid}/`,
```

Comment:
```suggestion
        hash: `#problems/${this.problem.alias}/show-run:${guid}`,
```

a esto le falta el alias, no?